client/pages/BookClubs/index.jsx

```diff
@@ -44,7 +45,10 @@ export default function BookClubs() {
         </div>
       </div>
     }
-      return <div className='most-visited-clubs'>LOGGED IN</div>
+      return <div className='all-my-clubs'>
+        LOGGED IN
+        <ClubCarousel />
+      </div>
```

Comment:
I think these changes roll back the PR from @jp-chang last night.

Is that intentional?

Reply:
No, strange it was not pulled from the main branch. Let me confirm shortly.